util/signal/signal_posix.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build linux darwin freebsd unix
+
+package signal
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
```

Comment:
```
 	"os"
 	"os/signal"
 	"runtime"
 	"syscall"

	log "github.com/sirupsen/logrus"
```

Reply:
Thanks for your patient, I have do some modify.